.github/workflows/check-resource-updates.yml

```diff
@@ -0,0 +1,105 @@
+name: Check Resource Updates
```

Comment:
@masiedu4 2 points on this, for now.
How do we test this?
Can we break the code into another file?

Reply:
For local testing, I came across https://nektosact.com/installation/homebrew.html

But eventually, we might need to merge this to a test branch perhaps to see how it works in prod. 

I updated the workflow to create a `Resource Updates Detected` issue instead of email when there is an updates, to reduce dependencies

@gbarros

Reply:
### Local testing with Act:
- Install Act: https://github.com/nektos/act#installation
- Create a .secrets file in the repo root:
- GITHUB_TOKEN=your_github_personal_access_token
- Run in terminal: `act workflow_dispatch -s GITHUB_TOKEN`